src/iter/mod.rs

```diff
@@ -30,6 +33,30 @@
 //! iterators work by first constructing a computation and then
 //! executing it.
 //!
+//! In addition to `par_iter()` and friends, some types offer other
+//! ways to create (or consume) parallel iterators:
+//!
+//! - Slices (`&[T]`, `&mut [T]`) offer methods like `par_split` and
+//!   `par_windows`, as well as various parallel sorting
+//!   operations. See [the `ParallelSlice` trait] for the full list.
+//! - Strings (`&str`) offer methods like `par_split` and `par_lines`.
+//!   See [the `ParallelString` trait] for the full list.
+//! - Various collections offer [`par_extend`], which grows a collection
+//!   given a parallel iterator. (You can use [`collect()`] to create a parallel
+//!   iterator.)
```

Comment:
Presumably you mean, "to create *a collection from* a parallel iterator."

Reply:
Yep.